doc/devel/index.rst

```diff
@@ -48,32 +100,26 @@ New contributors
          :class-row: sd-fs-5
 
          .. grid-item-card::
-            :link: request-a-new-feature
+            :link: contribute_code
             :link-type: ref
             :shadow: none
 
-            :octicon:`light-bulb;1em;sd-text-info` Request new feature
+            :octicon:`code;1em;sd-text-info` Contribute code
 
          .. grid-item-card::
-            :link: submitting-a-bug-report
+            :link: contribute_documentation
             :link-type: ref
             :shadow: none
 
-            :octicon:`bug;1em;sd-text-info` Submit bug report
+            :octicon:`note;1em;sd-text-info` Write documentation
 
          .. grid-item-card::
-            :link: contribute_code
+            :link: other_ways_to_contribute
             :link-type: ref
             :shadow: none
 
-            :octicon:`code;1em;sd-text-info` Contribute code
-
-         .. grid-item-card::
-            :link: contribute_documentation
-            :link-type: ref
-            :shadow: none
+            :octicon:`paper-airplane;1em;sd-text-info` Other ways to contribute
```

Comment:
I fundamentally argee.

I've made the current buttons exactly match the sections

![image](https://github.com/matplotlib/matplotlib/assets/2836374/cefe75e9-96f3-4064-b180-186d9a4e6567)

in https://matplotlib.org/devdocs/devel/contribute.html, because I believe it's valuable to have the same logical topics. I'm open to restructure that page and accordingly adapt the buttons. I propose to defer this to a separate PR because this restructuring is a non-trivial change and this PR is already large enough.